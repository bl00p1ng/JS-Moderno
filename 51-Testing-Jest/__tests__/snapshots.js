const client = {
    name: 'AndrÃ©s Felipe',
    credit: 500,
    type: 'Premium'
};

describe('Testing al cliente', () => {
    test('Comprobar nombre, saldo y tipo', () => {
        expect(client).toMatchSnapshot();
    });
});