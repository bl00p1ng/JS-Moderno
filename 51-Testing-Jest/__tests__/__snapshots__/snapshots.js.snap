// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Comprobar nombre, saldo y tipo 1`] = `
Object {
  "credit": 500,
  "name": "Andr√©s Felipe",
  "type": "Premium",
}
`;
